homeassistant/components/tesla/__init__.py

```diff
@@ -34,69 +40,150 @@
     extra=vol.ALLOW_EXTRA,
 )
 
-NOTIFICATION_ID = "tesla_integration_notification"
-NOTIFICATION_TITLE = "Tesla integration setup"
+
+@callback
+def _async_save_refresh_token(hass, config_entry, access_token, token):
```

Comment:
the naming is ambigious. One is the access token, so let's name the other `refresh_token`

Reply:
Done